### DISCUSSION TITLE
Option to hide some of the buttons e.g the sidebar button

### DISCUSSION THREAD

**Discussion by wilpoward** [Alex313031/thorium#350] [Ideas]

**Comment by gz83** [comment]
Which sidebar are you referring to?

@wilpoward

**Reply by wilpoward** [reply]
![image](https://github.com/Alex313031/thorium/assets/149069711/69a6747d-9f7e-4973-80e4-0d995f5d19f0)
these buttons

**Reply by gz83** [reply]
The avatar, sidepanel icon, and lab icon can be hidden in chrome://flags. Other icons cannot be hidden temporarily.

**Reply by wilpoward** [reply]
Ok thanks for that
